PROPOSAL.md

```diff
@@ -0,0 +1,70 @@
+# COMP 4350 Proposal Team - Spendr
+
+## Project Summary and Vision
+
+Spendr is designed to simplify group expenses by offering a comprehensive platform for users to collaboratively track and manage shared expenses, ensuring transparency, accuracy, and ease of settlements.
+
+## Core Features
+
+1. User Management
+2. Group Management
+3. Expense Categorization
+4. Expense Analytics
+5. Alerts and Notifications
+
+## Technologies
+
+[Potential technologies to be listed here.]
+
+## User Stories
+
+### Feature 1:  User Management
+
+- **User Story**: As a potential user, I want to register an account with Spendr to begin tracking my group expenses.
+  - **Acceptance Criteria**:
+    - Can register with email and password.
+    - Receive a confirmation email upon successful registration.
+
+- **User Story**: As a registered user, I want to log in to my Spendr account to access my groups and expenses.
+  - **Acceptance Criteria**:
+    - Can securely log in using email and password.
+    - Can log out of the account.
+
```

Comment:
Dont we a user store for adding individual expenses

Reply:
Yeah definitely, I added this under the second feature, though.

https://github.com/BarbzCodez/Spendr/pull/1/files/e3b999666c48e848a4b3c718aba380784e49e645#diff-3d0dee48edcfe8d2075980fdc47b991d02a098c79e8c20af3b56d181f2354d2cR40-R43